week1/react-exercises/src/2-guarantee/Guarantee.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+
+const Guarantee = () => {
```

Comment:
You are rendering a list sos make it plural 
```suggestion
const Guarantees = () => {
```